apps/question-bank/src/test/java/gov/cdc/nbs/questionbank/template/TemplateImporterTest.java

```diff
@@ -0,0 +1,134 @@
+package gov.cdc.nbs.questionbank.template;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.multipart.MultipartFile;
+import gov.cdc.nbs.questionbank.template.exception.TemplateImportException;
+import gov.cdc.nbs.questionbank.template.response.Template;
+
+@ExtendWith(MockitoExtension.class)
+class TemplateImporterTest {
```

Comment:
question:  how much of this could be moved into a feature test?

Reply:
A decent amount could be moved in to .feature tests, but we would still be mocking the response from Classic™ so it felt like leaving it to a unit test was sufficient